Billing docs revamp - Adjust links + remove old billing docs pages (#33489)
supabase/supabase: apps/docs/content/guides/platform/project-transfer.mdx

description: 'Transfer a project to another organization.'
---

> With the introduction of [Organization-based Billing](/docs/guides/platform/billing-on-supabase#organization-based-billing), we’re adding the ability to freely transfer projects between different organizations. Head to your [projects' general settings](https://supabase.com/dashboard/project/_/settings/general) to initiate a project transfer.

Reviewer: No longer need to mention the introduction of org based billing

```suggestion
You can freely transfer projects between different organizations. Head to your [projects' general settings](https://supabase.com/dashboard/project/_/settings/general) to initiate a project transfer.
```
